src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -142,7 +142,10 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
         agentUserId = getUserId(server)
       }
       if (jetbrainsUserId != agentUserId) {
-        logger.error("User id in JetBrains is different from agent")
+        if (agentUserId != null) {
```

Comment:
do we want to log `agentUserId is null`? idk, just asking

Reply:
It happens only when token was deleted on the website so in this case we should just prompt user to add valid account

Reply:
We should cove that case with a proper popup (like when user deny a keyring access) but that is rather separate issue. Please do not forgot to make a ticket for it @Sa1to :)